src/platform/platform_shared.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2020 Canonical, Ltd.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; version 3.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef MULTIPASS_PLATFORM_SHARED_H
+#define MULTIPASS_PLATFORM_SHARED_H
+
+#include <QString>
+
+namespace multipass::platform
+{
+QString interpret_general_hotkey(const QString& val);
```

Comment:
Why `_general_`?

Reply:
LOL, yeah indeed :smile: I'll remove that. 

When I started this was not in a separate header and was called by the actual platform functions to interpret the hotkey, so it was a way to distinguish.

Reply:
Done.